src/main/php/org/bovigo/vfs/vfsStreamAbstractContent.php

```diff
@@ -148,7 +148,7 @@ public function getType()
      * sets the last modification time of the stream content
      *
      * @param   int  $filemtime
-     * @return  vfsStreamContent
+     * @return  $this
```

Comment:
no, self may be used for static methods, but for instance appropriate one is ``$this``